content/zotero_intro_to.md

```diff
@@ -509,7 +508,7 @@ Make sure your Zotero data directory is included in the backups of your computer
 
 ![zotero][zotero] [Zotero Documentation : The Zotero Data Directory](https://www.zotero.org/support/zotero_data)
 
-**/!\\  Syncing is not a recommended strategy for backing up** . Indeed, the Zotero servers only store the most recent version of your library, and it takes just a single (possibly automatic) sync to change the server copy. Syncing is useful to get a copy of your library in multiple devices and to create group libraries, but it is different from backing up.
+**⚠️ Syncing is not a recommended strategy for backing up** . Indeed, the Zotero servers only store the most recent version of your library, and it takes just a single (possibly automatic) sync to change the server copy. Syncing is useful to get a copy of your library in multiple devices and to create group libraries, but it is different from backing up.
```

Comment:
to be left textual :point_right: find an Atom package supporting emoji when converting to PDF

Reply:
Ok